addons/base_vat/base_vat.py

```diff
@@ -227,11 +227,15 @@ def check_vat_ie(self, vat):
 
     # Mexican VAT verification, contributed by Vauxoo
     # and Panos Christeas <p_christ@hol.gr>
-    __check_vat_mx_re = re.compile(r"(?P<primeras>[A-Za-z\xd1\xf1&]{3,4})" \
-                                    r"[ \-_]?" \
-                                    r"(?P<ano>[0-9]{2})(?P<mes>[01][0-9])(?P<dia>[0-3][0-9])" \
-                                    r"[ \-_]?" \
-                                    r"(?P<code>[A-Za-z0-9&\xd1\xf1]{3})$")
+    __check_vat_mx_re = re.compile(br"(?P<primeras>[A-Za-z\xd1\xf1&]{3,4})" \
```

Comment:
Is this a whitespace change? Can you revert it, as it seems unrelated to this PR.

Reply:
It is a change of `r` to `br`, shall i revert this one?

Reply:
Yes please!